airflow/configuration.py

```diff
@@ -960,6 +960,14 @@ def get(  # type: ignore[override,misc]
         deprecated_section: str | None
         deprecated_key: str | None
 
+        option_description = self.configuration_description.get(section, {}).get(key, {})
+        if option_description.get("deprecated"):
+            deprecation_reason = option_description.get("deprecation_reason", "")
```

Comment:
```suggestion
        option_details = self.configuration_description.get(section, {}).get(key, {})
        if option_details.get("deprecated"):
            deprecation_reason = option_details.get("deprecation_reason", "")
```

nit, take it or leave it.

Reply:
We already have "configuration_description" all over the places :)

Reply:
fair enough 👍